djautotask/api.py

```diff
@@ -76,23 +76,26 @@ def get_cached_url(cache_key):
     return cache.get(cache_key)
 
 
-def get_api_connection_url():
-    return _get_connection_url('url')
+def update_cache(json_obj):
+    cache.set('url', json_obj['url'])
+    cache.set('webUrl', json_obj['webUrl'])
 
 
-def get_web_connection_url():
-    return _get_connection_url('webUrl')
+def get_api_connection_url(force_fetch=False):
+    return _get_connection_url('url', force_fetch)
```

Comment:
`url` is actually a poor cache key, and we shouldn't have used that. The django cache is shared so something else may try to use the `url` key accidentally in the future. Can you make this more specific? Like 'zoneUrl'?